### PR TITLE
iOS Improvements

### PR DESCRIPTION
**Pull request by rogeraabbccdd**
- Hide home indicator and defer system gestures on edges.
- Auto add `UIFileSharingEnabled` and `UISupportsDocumentBrowser` to `info.plist` after build.

### CODE REVIEW

File: TECHMANIA/Assets/Editor/BuildPostProcessor.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        if (buildTarget == BuildTarget.iOS)
+        {
+            string plistPath = path + "/Info.plist";
+            PlistDocument plist = new PlistDocument();
+            plist.ReadFromFile(plistPath);
+            PlistElementDict rootDict = plist.root;
+            rootDict.SetBoolean("UIFileSharingEnabled", true);
+            rootDict.SetBoolean("UISupportsDocumentBrowser", true);
+            File.WriteAllText(plistPath, plist.WriteToString());
```
Comment (macmillan333) [inline TECHMANIA/Assets/Editor/BuildPostProcessor.cs]: Can we do `plist.WriteToFile` so it's symmetric with `plist.ReadFromFile`?

**Review by macmillan333 — COMMENTED** [review]
One minor nitpick...

File: TECHMANIA/Assets/Editor/BuildPostProcessor.cs (hunk 1 above)
Comment (rogeraabbccdd) [reply TECHMANIA/Assets/Editor/BuildPostProcessor.cs]: This should be fixed in https://github.com/techmania-team/techmania/pull/9/commits/13c9f37c6d0aa2ccf904c9a039e733ab088bf606

**Review by macmillan333 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**